libensemble/tests/unit_tests/test_platform.py

```diff
@@ -17,6 +17,7 @@
     "gpu_setting_type": "option_gpus_per_task",
     "gpu_setting_name": "-g",
     "scheduler_match_slots": False,
+    "gpu_env_fallback": None,
```

Comment:
This should not be necessary, will default to None. I actually don't know if CUDA_VISIBLE_DEVICES would work on summit under these circumstances, but I can't currently test it.